source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -227,9 +251,11 @@ absl::Status ProcessorState::handleHeadersResponse(const HeadersResponse& respon
         return absl::OkStatus();
       }
       if (send_trailers_ && trailers_available_) {
-        // Trailers came in while we were waiting for this response, and the server
-        // is not interested in the body, so send them now.
-        filter_.sendTrailers(*this, *trailers_);
+        if (body_mode_ != ProcessingMode::FULL_DUPLEX_STREAMED) {
```

Comment:
Why we add this if check? Can you add some comments, please?

Reply:
done

Reply:
not done? I don't see the comment?

Reply:
The comments is added below:
// Skip doing this for FULL_DUPLEX_STREAMED body mode, as the trailers are already
// sent in this case.